yjit.c

```diff
@@ -1059,7 +1059,7 @@ rb_IMEMO_TYPE_P(VALUE imemo, enum imemo_type imemo_type)
 void
 rb_assert_cme_handle(VALUE handle)
 {
-    RUBY_ASSERT_ALWAYS(rb_objspace_markable_object_p(handle));
+    RUBY_ASSERT_ALWAYS(!rb_objspace_garbage_object_p(handle));
```

Comment:
You've inverted this assertion. Is this intentional? What does this assertion even do? 😅

Reply:
Yes this is intentional. `rb_objspace_markable_object_p` no longer exists. `rb_objspace_garbage_object_p` does (almost) the inverse of `rb_objspace_markable_object_p` by checking that `handle` is not a GC'd object.